arviz/tests/base_tests/test_data.py

```diff
@@ -456,6 +456,13 @@ def test_map(self, use):
         )
         assert np.allclose(idata_map.posterior.mu, idata.posterior.mu)
 
+    def test_repr_html(self):
+        idata = load_arviz_data("centered_eight")
+        html = idata._repr_html_()  # pylint: disable=protected-access
+
+        assert html is not None
+        assert "<div" in html
```

Comment:
I would make this assert more specific to check all required elements are present (at least partially). i.e. for html repr to work, we need our custom part (we could check for group names being present for example), the dataset repr should be present too (we could check for variable names of for Coordinate section title being present) and also style inherited from xarray (we could test a generic line/command from the style to assert is is correclty being appended to the repr)